handbook/product/surfacing_product_feedback.md

```diff
@@ -0,0 +1,30 @@
+# Surfacing feedback to the product team
+
+> If you're working with a specific customer on a bug or issue and need to directly involve a product engineering team, please refer to [engaging other teams](../ce/engaging-other-teams.md). 
+
+CEs, CSEs, sales, marketing, engineering, and other teammates that interact directly with Sourcegraph users and/or Sourcegraph itself should share product feedback with the product team. 
+
+We deeply value this feedback, so we make this process as frictionless for teammates as possible: 
+
+## Post the feedback in [`#feedback`](https://sourcegraph.slack.com/archives/C0W2E592M) on Slack
+
+You're welcome to tag teammates or teams, but that's not necessary – once the feedback is in [#feedback](https://sourcegraph.slack.com/archives/C0W2E592M) the product team will make sure it's routed correctly. 
+
+## When to create a GitHub issue
+
+If you have gotten a specific feedback requests that a customer(s) would like to track the progress of and a GitHub issue for the feedback [does not already exist](https://github.com/sourcegraph/sourcegraph/issues), you can also document this in a [new GitHub issue](https://github.com/sourcegraph/sourcegraph/issues/new/choose) to share with the customer. 
+
+Because a shareable issue is by definition public, you should not include any private customer information. You should still point to the customer by including a link to the customer in Salesforce.
+
+After you create a GitHub issue to share with a customer, please note in the body of the issue that `This issue was created so that a customer can track its progress` so the team is aware. Then you should share a link to this issue in `#feedback`. 
+
+## What if the feedback is [a possible exception]? 
+
+There are no exceptions to the above process! Please do post all product feedback to the #feedback channel – **this includes feedback that you feel might be: minor, not worth surfacing, too harsh, just a personal opinion, or from you rather than a customer**. All feedback helps us make Sourcegraph the best product possible. 
```

Comment:
In the recent CE/Product meeting, we were asked to share feedback in team channels as well when it was usability feedback. Can we abandon that in favor of all #feedback, all the time, with product handling processing?

Reply:
This is a good question, and a tension between "please put it in the right place to save time" and "it's a burden to know what the right place is" that I want to poll the product team on real quick.

Reply:
As noted in Slack, this should probably be decided in light of what is easier for the CE team.

Reply:
This will default to posting in the #feedback, so this line is correct